dvc/remote/local/__init__.py

```diff
@@ -48,6 +49,8 @@ class RemoteLOCAL(RemoteBASE):
     PARAM_CHECKSUM = "md5"
     PARAM_PATH = "path"
 
+    STATUS_VERIFICATION_DIR_SUFFIX = "unpacked"
```

Comment:
Similar to CHECKSUM_DIR_SUFFIX:
```suggestion
    UNPACKED_DIR_SUFFIX = ".unpacked"
```